core/store/LevelDBStore/LevelDBStore.go

```diff
@@ -62,7 +63,9 @@ func NewLevelDBStore(file string) (*LevelDBStore, error) {
 		it:                   nil,
 		header_index:         map[uint32]Uint256{},
 		block_cache:          map[Uint256]*Block{},
+		header_cache:         map[Uint256]*Header{},
```

Comment:
Rename  all such kinds of name to the unify Go style, the same to below several cases.